### ISSUE TITLE
Add Selection sort

### ISSUE THREAD

**Issue by vstrimaitis** [OpenGenus/cosmos#93]
Add the code for selectionsort in any language ( C, C++, Java, Go, Python or any other)

The code should be placed at code/sorting/selection_sort

**Comment by vstrimaitis** [comment]
Added implementation in C++: #85

**Comment by minijainsahab** [comment]
I will do it in java.

**Comment by vaibhavsingh97** [comment]
@vstrimaitis Can i work on this?

**Comment by n0zg** [comment]
I will do the java version

**Comment by beatfreaker** [comment]
Added implementation in JavaScript [#99](https://github.com/OpenGenus/cosmos/pull/99)

**Comment by vaibhavsingh97** [comment]
@n0zg Java version is already there 
I will do in python

**Comment by jainkeval** [comment]
will do it for c language

**Comment by RaiVaibhav** [comment]
@AdiChat added selection sort in ruby [151](https://github.com/OpenGenus/cosmos/pull/151)

**Comment by Guilhermeslucas** [comment]
I'm writing it in golang!

**Comment by shashwatjolly** [comment]
Has this been done in C++?

**Comment by RaiVaibhav** [comment]
@shashwatjolly u can check it by going throuh repositories

**Comment by AdiChat** [comment]
Yes, selection sort has been implemented in `C++` ([see](https://github.com/OpenGenus/cosmos/blob/master/code/sorting/selection_sort/selection_sort.cpp))

@shashwatjolly You can try it in other languages or look at other scopes of contribution 👍

**Comment by Ayush7-BIT** [comment]
I wanna add algorithm for it in java(Gssoc'20)kindly assign me this..
